### PR TITLE
docs: Add new LightGBM streaming docs

### PR DESCRIPTION
**Pull request by svotaw**
## What changes are proposed in this pull request?

Adjusted LightGBM docs for latest changes

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/a035ebd1b5101431ff1dd869f8ad32974985e78a/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 76 | 100 | 90 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54ebbd55-cb4a-4be9-b1b7-2000c32e5ea0) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/984f8eaa9be99c65607b154c4c1b808bf9d105f7/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 93 | 100 | 90 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a655df0a-558a-4d6c-a2f5-26fee1f24529) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/45532c48da27db6cc270d9864a64478dae78f8c4/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 90 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c11f605-b1bb-48f5-bfd7-1e9c6e868412) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: website/docs/features/lightgbm/about.md [hunk 1]
```diff
@@ -160,15 +161,59 @@ You can explicitly specify Execution Mode and MicroBatch size as parameters.
...
+from synapse.ml.lightgbm import LightGBMClassifier
+classifier = LightGBMClassifier(learningRate=0.3,
+                                numIterations=100,
+                                numLeaves=31)
+model1 = classifier.fit(train)
+
+classifier.learningRate = 0.4
+model2 = classifier.fit(train)
+```
+The *model2* call to *fit* doesn't resample the data and uses the same bin boundaries as *model1*.
```
Comment (mhamilton723) [inline website/docs/features/lightgbm/about.md:205]: might wanbt to change code refs to `backticks` inste3ad of *italics*

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Wee lil nit left otherwise lovely work!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/93c4b93af9fc7cba296f175c79977077dedce692/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 94 | 100 | 90 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f42dbc6e-c49a-4c04-9d67-56c0561f8732) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/cefddfafef36146bed682a9bdb05b71721fc2771/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 94 | 100 | 90 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a041ecc5-59c6-4db7-8291-8c9bce2f3149) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**